docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md

```diff
@@ -0,0 +1,153 @@
+
+#### Roadmap
+
+:::note  
+The following features are still in development and will be available in future releases.
+:::
+
+Before deploying the configurations to the cluster, users will be able to **review** which workload changes are going to be deployed by opening the review modal. This operation gives the developer a clear view of what is going to be released on the runtime environment, and it is especially important if the user is deploying on a production environment. 
+
+Moreover, in the new workflow the user will be able to **export Kubernetes configurations** by clicking on a download button.
+
+As of today, some **project customizations** can be performed only through the backoffice:
+
+- The repository default branch can be changed by updating the `project.repository.configurationBranch` property.
+- The project deploy strategy can be found in the `project.deploy.strategy` field for the global configuration that is inherited by all environments; to add environment specific settings you can patch the `project.environments[_].deploy.strategy`.
+
+In the future, these properties will be managed in the Console, inside the Project Settings page.
+
+## Activating the feature
+
+The Enhanced Project Workflow can be activated in different ways, depending on your needs.
+
+:::info
+In order to access the feature, please request the support of a Mia-Platform referent.
+To create a new Company or to activate feature toggles, please open a Service Request.
+:::
+
+The `ENABLE_CONF_GENERATION_ON_DEPLOY` feature toggle can be activated on a new project or Company to enable the enhanced project workflow. The feature toggle activates the new versioning system and the generation of Kubernetes configurations at the time of deployment. It can be enabled for a single project or for an entire Company, which means all projects in the Company will adopt the new workflow.
+
+If you already have a Company with some projects, and you do not wish to migrate them to the new approach just yet, you can choose the `ENABLE_CREATE_PROJECT_WITH_SAVE_CONFIG_ON_DEPLOY` feature toggle, to be activated on the Company. This feature toggle makes sure that all new projects in the Company will be created with the new workflow, while leaving the existing ones untouched. 
+
+:::note
+If both feature toggles are enabled, `ENABLE_CONF_GENERATION_ON_DEPLOY` will prevail.
+:::
+
+:::info
+In order to set up a new project, follow the [project creation guide](/console/project-configuration/create-a-project.mdx).
+:::
+
+## Manual adjustments on newly created projects
+
+After enabling the new workflow on one of your freshly created projects, some essential manual adjustments are necessary to make sure the project works correctly.
+
+First of all, the project model **must have set a default revision**: ask your Mia-Platform referent to check it, and in case it is missing, to add it manually. You can choose a custom name for your default revision, e.g. `main`.
```

Comment:
```suggestion
First of all, the Project Settings **must have set a default revision**: ask your Mia-Platform referent to check it, and in case it is missing, to add it manually. You can choose a custom name for your default revision, e.g. `main`.
```
add link to project settings section